enderio-base/src/main/resources/mixins.enderio.json

```diff
@@ -6,7 +6,6 @@
   "mixins": [
```

Comment:
Should this file be removed?

Reply:
I'm going to leave it there for now, right now we don't have mixins but until all features are implemented and accounted for I don't really want to remove this and have to put it back again. I've disabled mixins in my current fixes but left the json in.